fuse/README.md

```diff
@@ -0,0 +1,47 @@
+# Tachyon-FUSE
+Mount a Tachyon DFS using FUSE (**experimental**). 
+
+This project uses [jnr-fuse](https://github.com/SerCeMan/jnr-fuse) for FUSE on Java.
+
+## Requirements
+* JDK 1.8+
+* SBT 0.13.9+
+* libfuse 2.9.3+
+  (2.8.3 has been reported to also work - with some warnings)
+
+## Usage
```

Comment:
Added a "building" section to the README. At somepoint in the future all this info should be moved to a website page, I guess.